pkg/common/types.go

```diff
@@ -19,11 +20,10 @@ type Validator interface {
 }
```

Comment:
Please remove `ctx context.Context` from `Validate` if you are not using it

Reply:
Done